recipes/sphobjinv/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "sphobjinv" %}
+{% set version = "2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bskinn/sphobjinv/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 29c2287afe54682bdc6ff66337c36af7dda63b1198166bae7aa0185012e46dde
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - sphobjinv = sphobjinv.cli.core:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - attrs >=19.2
+    - certifi
+    - jsonschema >=3.0
+
+test:
+  requires:
+    - pip
+    - pytest
+    - pytest_check >=1.0,<1.1
+    - pytest-ordering
+    - pytest-timeout
+    - sphinx
+    - dictdiffer
+    - stdio-mgr
+  imports:
+    - sphobjinv
+  source_files:
+    - README.rst
+    - tox.ini
+    - requirements-dev.txt
+    - conftest.py
+    - tests/
+  commands:
+    - pip check
+    - pytest -k 'not readme'
```

This test is failing on Windows now, not sure why. But, if that is not a deal breaker we can merge this b/c the package is noarch.

---

I've ID-ed the problem and am narrowing down on a fix. Will be getting a new sdist to @anjos soon, which will hopefully provide an all-green test board.

---

(There's been a rabbit hole of Win vs Unix EOL silliness to dive down.)

---

Aha, nevermind -- this is a separate problem.

This should fix:

```suggestion
    - pytest -k "not readme"
```

---

(depending on whether YAML needs special escaping for double-quotes...can't recall......)